src/data/content/design-system/fondamenti/colori.yaml

```diff
@@ -0,0 +1,954 @@
+    kangaroo:
+      id: kangarooHero
+      noPadding: true
+      titleSr: Metadati e link per approfondire
+      navposition:
+        items:
+          - title: Ti trovi in
+            tooltip: "La sezione di Designers Italia cui appartiene questa pagina"
+            icon:
+              icon: "sprites.svg#it-folder"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Fondamenti #C
+            url: "../../" #M
+      otherInfo:
+        items:
+          - title: Rispettando
+            tooltip: "Norme e riferimenti che i contenuti di questa pagina ti aiutano a rispettare"
+            icon:
+              icon: "sprites.svg#it-pa"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Linee guida di design #C
+            url: "/norme-e-riferimenti/linee-guida-di-design" #M
+      personalInfo:
+         items: #C
+          - title: Sei
+            tooltip: "Se non ti riconosci in questi profili non preoccuparti: potrai scoprire informazioni utili per saperne di più. In alternativa, naviga la sezione superiore per approfondire il contesto"
+            icon:
+              icon: "sprites.svg#it-user"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Designer, Developer, Qui per curiosità #C
+          - title: Vuoi
+            tooltip: "Obiettivi di progettazione per cui sono utili i contenuti di questa pagina"
+            icon:
+              icon: "sprites.svg#it-flag"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Progettare e sviluppare l'interfaccia utente #C
+      tagsLabel: Parliamo di
+      color: secondary
+      icon:
+        icon: sprites.svg#it-horn
+        size: sm
+        color: secondary
+      tags: [Colori, Progettazione interfaccia]
+
+  sectionsEditorial :
+    - full: true
+      title:  Principi
+      text: I principi che guidano l'uso dei colori
+      background: null
+      components:
+        - name: TextImageCta
+          title: Rispetta i ruoli
+          noSpace: true
+          headingLevel: 3
+          text: |
+            Ciascun colore della palette ha un ruolo e funzione specifica. Ad esempio, il colore primario è associato ad elementi di azione mentre un colore neutro è riservato ad elementi di supporto o sfondi. Il rispetto di questi ruoli assicura coerenza visiva e chiarezza nella fruizione dei contenuti.
+
+        - name: TextImageCta
+          title: Applica con parsimonia
+          headingLevel: 3
+          noSpace: true
+          text: |
+            I colori contribuiscono alla creazione di una gerarchia visiva che mette in evidenza ciò che è davvero importante nel contesto della pagina. Non sono quindi elementi puramente decorativi, ma strumenti essenziali per orientare e semplificare l'esperienza utente attraverso lo stimolo visivo.
```

Comment:
Sottolineerei che il colore in ogni caso non può essere l'unico veicolo di una informazione. [Criterio 1.4.1](https://www.w3.org/Translations/WCAG21-it/#use-of-color) delle WCAG 2.1.